src/Microsoft.OData.Client/ALinq/TypeSystem.cs

```diff
@@ -223,6 +227,13 @@ static TypeSystem()
                 typeof(Date).GetProperty("Day", typeof(int)).GetGetMethod());
 
             Debug.Assert(propertiesAsMethodsMap.Count == 24, "propertiesAsMethodsMap.Count == 24");
+
+            aggregationMethodMap.Add(AggregationMethod.Sum, "sum");
+            aggregationMethodMap.Add(AggregationMethod.Average, "average");
+            aggregationMethodMap.Add(AggregationMethod.Min, "min");
+            aggregationMethodMap.Add(AggregationMethod.Max, "max");
+            aggregationMethodMap.Add(AggregationMethod.CountDistinct, "countdistinct");
+            aggregationMethodMap.Add(AggregationMethod.VirtualPropertyCount, "$count");

```

Comment:
Out of curiosity, why doesn't this use the string constants defined in `UriHelper`?